lib/superstring/dynamic-variables.js

```diff
@@ -0,0 +1,616 @@
+var faker = require('faker/locale/en'),
+    uuid = require('uuid'),
+
+    // locale list generated from: https://github.com/chromium/chromium/blob/master/ui/base/l10n/l10n_util.cc
+    LOCALES = ['af', 'am', 'an', 'ar', 'ast', 'az', 'be', 'bg', 'bh', 'bn', 'br', 'bs', 'ca', 'ceb', 'ckb', 'co', 'cs',
+        'cy', 'da', 'de', 'el', 'en', 'eo', 'es', 'et', 'eu', 'fa', 'fi', 'fil', 'fo', 'fr', 'fy', 'ga', 'gd', 'gl',
+        'gn', 'gu', 'ha', 'haw', 'he', 'hi', 'hmn', 'hr', 'ht', 'hu', 'hy', 'ia', 'id', 'ig', 'is', 'it', 'ja', 'jv',
+        'ka', 'kk', 'km', 'kn', 'ko', 'ku', 'ky', 'la', 'lb', 'ln', 'lo', 'lt', 'lv', 'mg', 'mi', 'mk', 'ml', 'mn',
+        'mo', 'mr', 'ms', 'mt', 'my', 'nb', 'ne', 'nl', 'nn', 'no', 'ny', 'oc', 'om', 'or', 'pa', 'pl', 'ps', 'pt',
+        'qu', 'rm', 'ro', 'ru', 'sd', 'sh', 'si', 'sk', 'sl', 'sm', 'sn', 'so', 'sq', 'sr', 'st', 'su', 'sv', 'sw',
+        'ta', 'te', 'tg', 'th', 'ti', 'tk', 'to', 'tr', 'tt', 'tw', 'ug', 'uk', 'ur', 'uz', 'vi', 'wa', 'xh', 'yi',
+        'yo', 'zh', 'zu'],
+
+    // paths for directories
+    DIRECTORY_PATHS = [
+        '/Applications',
+        '/bin',
+        '/boot',
+        '/boot/defaults',
+        '/dev',
+        '/etc',
+        '/etc/defaults',
+        '/etc/mail',
+        '/etc/namedb',
+        '/etc/periodic',
+        '/etc/ppp',
+        '/home',
+        '/home/user',
+        '/home/user/dir',
+        '/lib',
+        '/Library',
+        '/lost+found',
+        '/media',
+        '/mnt',
+        '/net',
+        '/Network',
+        '/opt',
+        '/opt/bin',
+        '/opt/include',
+        '/opt/lib',
+        '/opt/sbin',
+        '/opt/share',
+        '/private',
+        '/private/tmp',
+        '/private/var',
+        '/proc',
+        '/rescue',
+        '/root',
+        '/sbin',
+        '/selinux',
+        '/srv',
+        '/sys',
+        '/System',
+        '/tmp',
+        '/Users',
+        '/usr',
+        '/usr/X11R6',
+        '/usr/bin',
+        '/usr/include',
+        '/usr/lib',
+        '/usr/libdata',
+        '/usr/libexec',
+        '/usr/local/bin',
+        '/usr/local/src',
+        '/usr/obj',
+        '/usr/ports',
+        '/usr/sbin',
+        '/usr/share',
+        '/usr/src',
+        '/var',
+        '/var/log',
+        '/var/mail',
+        '/var/spool',
+        '/var/tmp',
+        '/var/yp'
+    ],
+
+    // generators for $random* variables
+    dynamicGenerators = {
```

Comment:
remove `dynamicGenerators` assignment and directly export the object.

Reply:
We need to have the `dynamicGenerators` assignment because we are accessing `$randomDirectoryPath` inside `$randomFilePath`.

See: https://github.com/postmanlabs/postman-collection/pull/904/files/2029aa6b162b3662a3ad79291957cf89aa68a6b5#diff-b3d7c12136fa69f68a89843eb1b062a3R149

Reply:
Got it, there's a workaround but it's fine. No change required.